pkg/cloud/cloudcheck/cloudcheck_processor.go

```diff
@@ -114,7 +113,8 @@ func checkStorage(
 	}
 
 	buf := make([]byte, chunkSize)
-	_, _ = crypto_rand.Read(buf)
+	//lint:ignore SA1019 deprecated
```

Comment:
Hmm. A little odd that we read a random integer then throw away the result, isn't it? Clearly it was that way before, but I'm wondering what purpose that serves.

Reply:
@dt authored this code; he is best equipped to explain the purpose. it's not thrown away though - the bytes are written to external storage on line 130

Reply:
Ah, I see. I was confused by the function signature.